server/src/test/java/io/crate/integrationtests/KillIntegrationTest.java

```diff
@@ -95,7 +95,7 @@ private void assertGotCancelled(final String statement, @Nullable final Object[]
                 future.get(10, TimeUnit.SECONDS);
             } catch (Throwable exception) {
                 exception = SQLExceptions.unwrap(exception); // wrapped in ExecutionException
-                assertThat(exception, instanceOf(JobKilledException.class));
+                assertThat(exception, anyOf(instanceOf(JobKilledException.class), instanceOf(InterruptedException.class));
```

Comment:
This cover the following case https://github.com/crate/crate/blob/6814214331fdaaa5807d33804081c1528828d0db/libs/dex/src/test/java/io/crate/testing/FailingBatchIterator.java#L46

The test used to match against `SqlActionException` which wrapped all underlying exception. Now since `SqlActionException` is gone we need to match these cases one-by-one.